src/storage/blockchain/did-resolver-readonly-storage.ts

```diff
@@ -74,7 +75,14 @@ export class DidResolverStateReadonlyStorage implements IStateStorage {
       this.resolverUrl,
       opts
     );
-    const { info } = this.getIden3StateInfo2023(didDocument);
+    const { info, published } = this.getIden3StateInfo2023(didDocument);
+    if (!info && !published) {
+      throw new Error(VerifiableConstants.ERRORS.STATE_DOES_NOT_EXIST);
+    }
+    if (info) {
```

Comment:
is it possible that info is undefined?
If so, I guess we need to throw error
Otherwise { ...info } returns just empty object